catalog/app/components/Preview/loaders/Notebook.tsx

```diff
@@ -53,5 +54,9 @@ export const Loader = function WrappedNotebookLoader({
   handle,
   children,
 }: NotebookLoaderProps) {
-  return <NotebookLoader {...{ handle, children }} />
+  return (
+    <React.Suspense fallback={<M.CircularProgress />}>
```

Comment:
hm, probably we should call `children` with `AsyncResult.Pending()` instead of showing the placeholder directly

Reply:
Done